### ISSUE TITLE
Cocoao Pods support for tvOS

### ISSUE THREAD

**Issue by crashoverride777** [jrendel/SwiftKeychainWrapper#58]
Please can you integrate this project in cocoa pods for tvOS. This helper works flawlessly on tvOS but without the cocoaPod integration its a bit of a hassle when having a universal iOS and tvOS project.

**Comment by jfgrang** [comment]
Yes please approve the PR : https://github.com/jrendel/SwiftKeychainWrapper/pull/129
